src/platform_impl/macos/window.rs

```diff
@@ -100,8 +100,13 @@ unsafe fn create_view(
 ) -> Option<(IdRef, Weak<Mutex<CursorState>>)> {
     let (ns_view, cursor_state) = new_view(ns_window);
     ns_view.non_nil().map(|ns_view| {
+        // The default value of `setWantsBestResolutionOpenGLSurface_` was `false` until
+        // macos 10.14 and `true` after 10.5, we should set it to `YES` o `NO` to avoid
+        // always the default system value in favour of the user's code
```

Comment:
Nit:

```suggestion
        // The default value of `setWantsBestResolutionOpenGLSurface:` was `false` until
        // macos 10.14 and `true` after 10.15, we should set it to `YES` or `NO` to avoid
        // always the default system value in favour of the user's code
```